docs/content/contributing/bicep/bicep-contribution-flow/owner-contribution-flow.md

```diff
@@ -0,0 +1,180 @@
+
+{{< /hint >}}
+
+---
+
+### 1. Owner Activities and Responsibilities
+
+<!-- TODO: Rephrase sections if required -->
+Familiarise yourself with the responsibilities as **Module Owner** outlined in [Team Definitions & RACI](https://azure.github.io/Azure-Verified-Modules/specs/shared/team-definitions/#module-owners) and [Module Owner Responsibilities](https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/brm-issue-triage/#module-owner-responsibilities) in the [BRM Issue Triage](https://azure.github.io/Azure-Verified-Modules/help-support/issue-triage/brm-issue-triage/).
+
+1. Create GitHub teams as outlined in [SNFR20](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only) and add respective parent teams:
+
+Segments:
+
+- `avm-res-<RP>-<modulename>-module-owners-bicep`
+- `avm-res-<RP>-<modulename>-module-contributors-bicep`
+
+Examples:
+
+- `avm-res-compute-virtualmachine-module-owners-bicep` and added `avm-technical-reviewers-bicep` as parent.
+- `avm-res-compute-virtualmachine-module-contributors-bicep` and added `avm-module-contributors-bicep` as parent.
+
+If a secondary owner is required, add the secondary owner to the `avm-res-<RP>-<modulename>-module-contributors-bicep` team.
+
+[!NOTE]
+Only fulltime Microsoft employees can be added at this time.
+
+{{< hint type=info >}}
+
+Once the teams have been created the AVM Core Team will review the team name and parent team membership for accuracy. A notification will automatically be sent to the AVM Core Team to notify them that their review needs to be completed.
+
+{{< /hint >}}
+
+1. Add teams to `CODEOWNERS` file as outlined in [SNFR20](https://azure.github.io/Azure-Verified-Modules/specs/shared/#id-snfr20---category-contributionsupport---github-teams-only).
+
+2. Make sure your module has been tested before raising a PR. This can happen in your or in another module contributor's own environment - if any. Also, because once a PR is raised, a GitHib workflow pipeline is required to be run successfully before the PR can be merged. This is to ensure that the module is working as expected and is compliant with the AVM specifications.
+
+3. Ensure that the module(s) you own are compliant with the AVM specifications and are working as expected. The following specifications are to be considered and where `Owner` is mentioned explicitly:
+
+| ID | Specification
```

Comment:
Remove "https://azure.github.io" from all the links on this page, and let the links be relative, starting with "/Azure-Verified-Modules/....."